crates/sui-graphql-rpc/src/types/checkpoint.rs

```diff
@@ -239,6 +242,34 @@ impl Checkpoint {
         }))
     }
 
+    /// Look up the latest checkpoint sequence number in the database. This should be used to set
```

Comment:
(Based on the outcome of the discussion around execution and dry run, we may no longer need these).